### DISCUSSION TITLE
kprobe can not get the syscall args

### DISCUSSION THREAD

**Discussion by guozijing** [cilium/ebpf#1561] [Q&A]
Bellow is my code to get the args for syscall sched_setaffinity, but I can not get the right args for this syscall. can anyone help me?
The second parms for sched_setsffinity should be 16 in my example commands, "taskset -c 1 ls", but i will get a large number from ebpf.

**Comment by ti-mo** [comment]
@guozijing This doesn't look very related to ebpf-go itself, you may have more luck with this kind of question on StackOverflow.
